dex/price-discovery/src/events.rs

```diff
@@ -0,0 +1,60 @@
+elrond_wasm::imports!();
+
+#[elrond_wasm::module]
+pub trait EventsModule {
+    #[event("depositExtraRewardsEvent")]
+    fn deposit_extra_rewards_event(
+        &self,
+        #[indexed] block: u64,
+        #[indexed] caller: &ManagedAddress,
+        amount: &BigUint,
+    );
+
+    #[event("depositEvent")]
+    fn deposit_event(
```

Comment:
for `deposit` and `withdraw` events would be nice to have also the amounts for launched tokens and accepted tokens

Reply:
Done.